README.md

```diff
@@ -55,14 +55,8 @@ application behaviour.
 
   `export PYTHON_PACKAGES="./python/requirements.txt ./builder/requirements.txt"`
 
-* `PYTHON_VERSION`: Required when `PYTHON_PACKAGES` is defined. 
-  Version of Python to use when running python
-  dependency scan.  
-  Format of version number follows the same convention of Alpine's 
-  [APK package manager](https://wiki.alpinelinux.org/wiki/Package_management#Advanced_APK_Usage)   
-  Example:
-
-  `PYTHON_VERSION=~3.8.10`
+* `PYTHON_BUILDER`: Required. Image to use for generatin Python 
+  dependencies.
```

Comment:
An example here would be good. Also a hint towards how to find images for other versions if it's the image that was just removed from the dockerfile.